change/@fluentui-react-storybook-addon-ee7ab7c2-7aae-4b6f-8e9f-700123d875f7.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
this is actually a breaking change as we bump minimun SB support, please update type